### PR TITLE
making headlines more consistent with hierarchy and styling

### PR DESCRIPTION
**Pull request by cpham**
#312 making headlines more consistent with hierarchy and styling

### CODE REVIEW

**Comment by liz-shinn** [comment]
I have a GH issue open for some of these fixes – are these fixes related to that issue? GH link here: https://github.com/keep-network/website/issues/312

**Comment by liz-shinn** [comment]
I think that the h1 and h2 classes are looking much more consistently applied. However, they don't match the Figma design guidelines for the different classes of type hierarchy, [here in Figma](https://www.figma.com/file/ARAMFLMOPal0JewjvlZGhX/Keep-Design-System?node-id=1097%3A0).

Screen Shot 2020-11-03 at 12 41 09 PM

**Comment by liz-shinn** [comment]
Versions of privacy pages in Figma files:

Privacy Policy (doesn't have all page text, but has the heading classes applied) https://www.figma.com/file/lw986GpKJjWATIGVHcyMTg/Keep-Website?node-id=1138%3A1832

Keep Network Terms of Use (doesn't have all page text, but has the heading classes applied)
https://www.figma.com/file/lw986GpKJjWATIGVHcyMTg/Keep-Website?node-id=1115%3A1435

**Comment by Shadowfiend** [comment]
Next push should get us a preview link.

**Comment by cpham** [comment]
@liz-shinn take a look now, should be consistent with figma

**Comment by liz-shinn** [comment]
Checking it now! 👀

**Comment by liz-shinn** [comment]
Sweet - checked out the preview link and it matches Figma and h1 & h2 look great. I'm wondering if the page date shouldn't be applied as h3 or does that break things elsewhere in the site?

Screen Shot 2020-11-04 at 10 45 08 AM

**Comment by liz-shinn** [comment]
Because it is h3 here... hmm is this because the font color is different in the date instance?

Screen Shot 2020-11-04 at 10 45 08 AM

**Comment by Shadowfiend** [comment]
Date as `p` is good (the date is not a heading---easy test on this is “would it show up in a table of contents?”). Are you concerned about the styling though?

File: src/css/legal.scss [hunk 1]
```diff
@@ -6,41 +6,70 @@
...
+    }
+
+    h4 {
+        font-family: 'Work Sans', sans-serif;
+        font-size: 16px;
+        font-weight: bold;
+        line-height: 25px;
+        letter-spacing: -.5px;
+        margin: 1rem 0;
+    }
```
Comment (Shadowfiend) [inline src/css/legal.scss:44]: Not necessarily for right this moment, but if these are all design system settings I wonder if we should look into pulling them to `typography.scss`. At a glance, they're different values, which is interesting---would love to dig in a bit and see if the issue is that the design system is oriented around regular content and the current top-level settings in typography.scss are rooted in the landing page—i.e., that the exception is the landing page, not the legal page.

Anyway, something to chew on for future updates.

File: src/css/legal.scss [hunk 2]
```diff
@@ -6,41 +6,70 @@
...
         }
 
-        p:first-of-type {
-            margin-top: 3.5em;
+        .date {
+            font-family: 'Work Sans', sans-serif;
+            font-size: 24px;
+            line-height: 32px;
+            font-weight: 500;
+            letter-spacing: -.5px;
```
Comment (Shadowfiend) [inline src/css/legal.scss:57]: I do wonder if it makes sense to set `letter-spacing: -.5px` across the site… May be worth the experiment.

File: src/css/legal.scss [hunk 3]
```diff
@@ -6,41 +6,70 @@
...
+            color: #4c4c4c;
```
Comment (Shadowfiend) [inline src/css/legal.scss:58]: We have this defined as variable [`$grey-70`](https://github.com/keep-network/website/blob/7f37443e928407d4ddea03d8f46488fe8e2bd8a4/src/css/vars.scss#L14), would be worth moving to that in a follow-up PR.

**Review by Shadowfiend — APPROVED** [review]
Left a few comments, but nothing that's blocking the PR merge. Let's ship it :shipit:

**Comment by liz-shinn** [comment]
>Date as p is good (the date is not a heading---easy test on this is “would it show up in a table of contents?”). Are you concerned about the styling though?

Nope styling looks fine, I just have it as an instance of h3 in the design files since it's the same as an h3 heading. Was wondering if it shouldn't match here. I think my conceptual model of headings is based more on their relative size and less their syntax, which works fine in figma land but sounding like doesn't translate correctly into not-figma land

**Pull request merged**